pkg/gadgets/internal/socketenricher/bpf/sockets-iter.bpf.c

```diff
@@ -31,6 +31,8 @@ insert_socket_from_iter(struct sock *sock, struct task_struct *task)
 	__builtin_memcpy(&socket_value.task, task->comm, sizeof(socket_value.task));
 	socket_value.mntns = (u64) task->nsproxy->mnt_ns->ns.inum;
 	socket_value.sock = (__u64) sock;
+	if (socket_key.family == AF_INET6)
```

Comment:
What do you think about removing this conditional as `skc_ipv6only` should be false when `socket_key.family != AF_INET6`? Do you prefer to keep this just in case?

Reply:
Yes, it should work. Let me try.